Resources/public/js/views/services/plugins/ez-savedraftplugin.js

```diff
@@ -61,9 +61,9 @@ YUI.add('ez-savedraftplugin', function (Y) {
                 },
             });
             if ( isNew ) {
-                this._createContent(e.fields);
+                this._createContent(e.fields, e.serverSideErrorCallback);
```

Comment:
related to the other BC comment (https://github.com/ezsystems/PlatformUIBundle/pull/672/files#r78738602), I think you should not add a parameter to these methods. I realize, I already this issue and to avoid that, `this._callback` was added so you should probably adopt a similar strategy here and in the other plugin.